### DISCUSSION TITLE
How dangerous it is to share public keys and share information which keys is used to encrypt file?

### DISCUSSION THREAD

**Discussion by aca** [FiloSottile/age#426] [Q&A] [answered]
Hi, 
I'm thinking of sharing secrets in team based on age. But In order to identify who is the owner of the secret, re-encrypt, I am trying save information about plain public keys used to encrypt file.

There's a mention about this in README. But I'm not a security expert and not sure how much dangerous it is to share plain recipients(public key).
> Note that SSH key support employs more complex cryptography, and embeds a public key tag in the encrypted file, making it possible to track files that are encrypted to a specific public key.

Sorry for stupid question.

**Comment by FiloSottile** [comment, marked as answer]
There are no stupid questions, only incomplete docs :)

There is no risk in storing public keys with encrypted files, except the obvious ones: it will be possible to know what files were encrypted to what keys, and it will be possible to re-encrypt different files to those keys.

**Reply by aca** [reply]
Thanks! Always love to see your work!
